docs/README.md

```diff
@@ -581,7 +581,12 @@ onTransaction = transactionCallback
 
 > **The ‘on-open’ parameter specifies the callback function that runs when the button is clicked, before the dialog opens.**
 
-?> This parameter is optional. Default value is empty. Possible values are any defined function.
+#### *callback* arguments
+
+- **expectedAmount**: *number* - the amount set
+- **address**: *string* - your address
+- **paymentId**: *string* - the paymentId
+- **to**: *string* - the destination address
```

Comment:
I think we might have made some mistakes when implementing this change originally.

These should be:

`amount` (not `expectedAmount`)
`paymentId`
`to` (no `address`)

So my suggestion is to change it to these ^ for now and we'll make a new PR for fixing these up to be in line with the rest of the site.